virtcontainers/kata_agent.go

```diff
@@ -252,6 +253,11 @@ func (k *kataAgent) configure(h hypervisor, id, sharePath string, builtin bool,
 		k.proxyBuiltIn = true
 	}
 
+	caps := h.capabilities()
+	if !caps.is9pSupported() {
```

Comment:
Please add some comments here, making clear that if 9p is not supported, we don't want to create any shared directory, which is why we're returning right here.